image/templates/helm/shared/templates/02-scanner-v4-08-indexer-service.yaml

```diff
@@ -0,0 +1,48 @@
+{{- include "srox.init" . -}}
+{{- if ._rox.scannerV4._indexerEnabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: scanner-v4-indexer
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "srox.labels" (list . "service" "scanner-v4-indexer") | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "service" "scanner-v4-indexer") | nindent 4 }}
+spec:
+  selector:
+    app: scanner-v4-indexer
+  clusterIP: None
+  ports:
+  - name: grpcs-scanner-v4-indexer
+    port: 8443
+    targetPort: 8443
+  {{- if ._rox.scannerV4.exposeMonitoring }}
+  - name: monitoring
+    port: 9090
+    targetPort: monitoring
+  {{- end }}
+
+{{- if ._rox.env.istio }}
+---
+
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: scanner-v4-indexer-internal-no-istio-mtls
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "srox.labels" (list . "destinationrule" "scanner-v4-indexer-internal-no-istio-mtls") | nindent 4 }}
+  annotations:
+    stackrox.io/description: "Disable Istio mTLS for port 8443, since StackRox services use built-in mTLS."
+    {{- include "srox.annotations" (list . "destinationrule" "scanner-v4-indexer-internal-no-istio-mtls") | nindent 4 }}
+spec:
+  host: scanner-v4-indexer.{{ .Release.Namespace }}.svc.cluster.local
+  trafficPolicy:
+    portLevelSettings:
```

Comment:
do we need to do this for the metrics port, too?

Reply:
tbh, I am not familiar with istio. I will try to come up with an answer.

Reply:
I don't think we need the metrics port. Just 8443, which is what we have now